docs/features.md

```diff
@@ -0,0 +1,47 @@
+## Shop-easy platform features
+
+There are two user roles in our platform. They are
+    - Admin User
+    - Student User
+
+### Admin User:
+
+Admin has administrative previliges and responsible for pantry management such as Inventory management and order tracking etc.,
+
+### Student User:
+
+Student can access the pantry platform for convinent and user friendly purchases of the listed items.
+
+### Admin features:
```

Comment:
Admin Features:
View Reviews and Suggestions: Admin user Should be able to see the suggestion/review messages from student users. Just a simple GET/Fetch that displays the suggestions entered by students.

Reply:
Addressed